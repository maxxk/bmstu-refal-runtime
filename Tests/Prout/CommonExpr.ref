$ENTRY Go
{
	= <Prout A S D Daasd 'check' ASD 'qweq' 2 'sad' (('c' "asdasd") 'sdas' (1 "Ha-ha-ha" 2 3)) (12 3 ) () () ('asd') "Hello" Aaa>
}
