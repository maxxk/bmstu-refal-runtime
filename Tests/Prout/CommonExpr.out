A S D Daasd checkASD qweq2 sad((casdasd )sdas(1 Ha-ha-ha 2 3 ))(12 3 )()()(asd)Hello Aaa 
